docs/reference/commandline/tag.md

```diff
@@ -35,7 +35,7 @@ components may contain lowercase letters, digits and separators. A separator
 is defined as a period, one or two underscores, or one or more dashes. A name
 component may not start or end with a separator.
 
-A tag name must be valid ASCII and may contain lowercase and uppercase letters,
+A tag name must be valid ASCII and may contain lowercase letters,
```

Comment:
@mattray It looks like these errors improved since docker 17.03; running on docker 17.05, I get;

```bash
$ docker image tag busybox mattray/R:testing
Error parsing reference: "mattray/R:testing" is not a valid repository/tag: invalid reference format: repository name must be lowercase


$ docker image tag busybox foobar@sha256:4296639ebdf92f035abf95fee1330449e65990223c899838283c9844b1aaac4c
refusing to create a tag with a digest reference
```